example.env

```diff
@@ -1,211 +0,0 @@
-# General Config
-# NOTE: The service_role key is required as an authorization header for /admin endpoints
-
-GOTRUE_JWT_SECRET="CHANGE-THIS! VERY IMPORTANT!"
```

Comment:
/nit When this file is re-introduced, do add the example env vars for both `example.docker.env` and `example.env` 



e.g. 
```
GOTRUE_SMS_MSG91_API_KEY=""
GOTRUE_SMS_MSG91_SECRET=""
GOTRUE_SMS_VONAGE_FROM=""
```

etc